plugin

```diff
@@ -333,6 +333,7 @@
   "Villhellm/lovelace-clock-card",
   "Voxxie/lovelace-jumbo-card",
   "vpdchart/vpdchart-card",
+  "WaresWichall/hass-anycubic_card",
```

Comment:
<https://github.com/WaresWichall/hass-anycubic_card/issues/2> That is the same as not having issues enabled, which is not accepted.